CMakeLists.txt

```diff
@@ -15,8 +15,13 @@ if (CMAKE_CROSSCOMPILING)
         set(VULKAN_PREBUILT_BIN_GLSLC sdk/windows/Bin/glslc.exe)
         set(VULKAN_PREBUILT_BIN_SPIRV_CROSS sdk/windows/Bin/spirv-cross.exe)
     elseif (${CMAKE_HOST_SYSTEM_NAME} MATCHES "Linux")
-        set(VULKAN_PREBUILT_BIN_GLSLC sdk/linux/bin/glslc)
-        set(VULKAN_PREBUILT_BIN_SPIRV_CROSS sdk/linux/bin/spirv-cross)
+        if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
+            set(VULKAN_PREBUILT_BIN_GLSLC sdk/linux-aarch64/bin/glslc)
+            set(VULKAN_PREBUILT_BIN_SPIRV_CROSS sdk/linux/bin/spirv-cross)
```

Comment:
I think this path needs aarch64?

Reply:
you are correct. somehow missed that. thanks!